pkgs/top-level/python-packages.nix

```diff
@@ -1809,7 +1809,9 @@ in {
   # alias
   elasticsearchdsl = self.elasticsearch-dsl;
 
-  elasticsearch-curator = callPackage ../development/python-modules/elasticsearch-curator { };
+  elasticsearch-curator = callPackage ../development/python-modules/elasticsearch-curator {
+    click = callPackage ../development/python-modules/click { version = "6.7"; };
```

Comment:
We can't do that for Python packages as multiple versions of the same package can't be in `$PYTHONPATH`.

Reply:
We could use `packageOverrides` if elasticsearch-curator was only an application. However, according to https://github.com/elastic/curator#curator-api-documentation it also has an API.

Reply:
:( That's the end of my python knowledge, tbh. I only use the module, which use elastic-curator as an application: https://github.com/NixOS/nixpkgs/blob/be1c03ddaf867e9a58499cd790d5cd72cffc6fca/nixos/modules/services/search/elasticsearch-curator.nix#L89 .. So I'd be fine with packageOverride or simply wrapping the binary, but I don't know what other usecases for this package that would destroy.